letsencrypt/client.py

```diff
@@ -221,11 +221,16 @@ def _obtain_certificate(self, domains, csr):
 
         logger.debug("CSR: %s, domains: %s", csr, domains)
 
-        authzr = self.auth_handler.get_authorizations(domains)
+        if not self.config.skip_authz:
+            authzrs = self.auth_handler.get_authorizations(domains)
+            certr_url = authzrs[0].new_cert_uri
```

Comment:
The `# TODO: acme-spec #90` comment was removed. While the referenced issue was closed by https://github.com/letsencrypt/acme-spec/commit/7aac185b6c05b3665d682f9efbbd8bac70effc96, I still think we should leave a comment here on why it's enough to look at `[0]`. In fact I would actually check that all authorizations have the same URI and `logger.warn` or even raise an error otherwise.

Please use `new_cert_uri` name.

You can inline `authzrs`: `self.auth_handler.get_authorizations(domains)[0].new_cert_uri`